src/java/org/apache/cassandra/db/Directories.java

```diff
@@ -942,6 +942,7 @@ private void filter()
                 switch (type)
                 {
                     case TXN_LOG:
+                    case OBSOLETE:
```

Comment:
as far as I gathered, the rest of the changes from https://github.com/riptano/bdp/pull/18705/files were ported already (either to Cassandra or moved to cndb during remote fs port).